app/mm-redux/store/helpers.ts

```diff
@@ -1,42 +1,38 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
+
 import {combineReducers} from 'redux';
-import {General} from '../constants';
 import reducerRegistry from './reducer_registry';
-import {enableBatching, Action, Reducer} from '@mm-redux/types/actions';
-export const offlineConfig = {
-    effect: (effect: Function, action: Action) => {
-        if (typeof effect !== 'function') {
-            throw new Error('Offline Action: effect must be a function.');
-        } else if (!('meta' in action && action.meta && action.meta.offline.commit)) {
-            throw new Error('Offline Action: commit action must be present.');
-        }
-
-        return effect();
-    },
-    discard: (error: Error, action: Action, retries: number) => {
-        if ('meta' in action && action.meta && action.meta.offline.hasOwnProperty('maxRetry')) {
-            return retries >= action.meta.offline.maxRetry;
-        }
-
-        return retries > 10;
-    },
-};
-
-export function createReducer(baseState: any, ...reducers: Reducer[]) {
+import {enableBatching, Reducer} from '@mm-redux/types/actions';
+import AsyncStorage from '@react-native-community/async-storage';
+
+/* eslint-disable no-console */
```

Comment:
Exclusion could be moved to the relevant `console.log` line in the `catch` block later?

Reply:
I do prefer to have the console log there for debugging purposes

Reply:
👍